SearchList.js

```diff
@@ -105,33 +79,33 @@ export default class SearchList extends Component {
     this.tmpSource = []
   }
 
-  static getSectionData (dataBlob, sectionID) {
+  static getSectionData(dataBlob, sectionID) {
```

Comment:
Our code style follows the [standardjs](https://standardjs.com/), fix the code format please, thx.

Reply:
ok ok I will